tortoise/README.md

```diff
@@ -0,0 +1,167 @@
+Tortoise is used to make decisions on the blocks that will be applied to the ledger. The protocol is asynchronous and makes progress based on the received data, without timing assumptions.
+
+## Core concepts
+
+#### Block
+
+From the tortoise perspective it is important that the block has a unique ID and belongs to some layer.
+
+In the rest of the system block is a wrapper for transactions and rewards. So once tortoise decides on a block - related transactions and rewards will be applied to the ledger.
```

Comment:
q: are we 100% sure that rewards must be part of a block's content? We have discussed this many times in the past but I'm not sure what's the latest consensus on this question.

Reply:
a: Yes, we're 100% sure we need explicit rewards under UCB. The reason is that blocks under this model have no reference to the smeshers that provided input for them. This information is known at the time of creating the block but it's ephemeral, so can't be used for making local decisions (crediting some account).

The rewards in the block will not specify an explicit amount: when constructing the block we can't know the exact fees collected - the transactions must be executed to know for sure. Instead, the rewards will specify a weight per coinbase account and the full layer reward (incl. fees) will be distributed between the accounts by weight.

This approach will also make it easier to provide rewards for other roles than layer committee member (the ones that create ballots and proposals), like Hare participants and one day validators. For now, we'll only reward layer committee members, though.